.devcontainer/Dockerfile

```
@@ -22,6 +24,13 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
         # tedge dependencies
         mosquitto
 
+# Install gh utility
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+    && sudo apt update \
+    && sudo apt install gh -y
```

```suggestion
    && sudo apt install -y gh
```

Purely cosmetic change, but it reads a bit more conventional to other `apt-get` installs.